docs/aad-application-configuration.md

```diff
@@ -0,0 +1,115 @@
+
+# AAD application configuration
+
+The deployed OpenShift cluster needs a valid AAD application and service principal to call back into the Azure API,
+in order to enable AAD authentication. There are a few options here:
+
+* Create application using Azure Portal and configure all values there.
+
+* Create application using CLI and later configure permission in the Azure portal.
+
+* Create application using CLI and approve permission in the Azure portal
+
+To specify your own application for OpenShift to use, add AAD flags to the `az openshift create` command:
+
+```azurecli-interactive
+OSA_CLUSTER_NAME=mjudeikis-prod1
```

Comment:
sanitize to something without your name :)

Reply:
I was this close to getting this in! :dancer: